### ISSUE TITLE
Mods

### ISSUE THREAD

**Issue by Gengar-lab** [MattZ45986/Bombsquad_mods#5]
@MattZ45986 Have all my mods completed?

**Comment by MattZ45986** [comment]
@Benefit-Zebra, yeah, I'm sorry it's been taking so long.  I've got some other projects I've been working on, so the mods were put on hold.  When I have some more time in the next few weeks I hope to complete the mods you suggested because I'm still excited about them, especially the powerups. 
If there is anything you'd like me to work on particularly, let me know: I'll finish the ones you want most first.
Then, if you share them, test them, and other people like it, I'll put it on the official mod-manager since the owner made me a collaborator!

**Comment by Gengar-lab** [comment]
@MattZ45986 Some of your mods only support FFA , Please make them suitable for Team Playing Also.

**Comment by Gengar-lab** [comment]
@MattZ45986 Can you Make a Powerup that if we take it our health would increase.

Another : If we took that powerup and die , the position where we died, bombs would erupt from that place

**Comment by MattZ45986** [comment]
For that first powerup, you'll have to distinguish it from a health pack. Right now there are no differences. Second one, yeah I could do that. Give me a day or two.

**Comment by Gengar-lab** [comment]
@MattZ45986 Take how much time you want.

But Please Inform Me when you finish it.

**Comment by MattZ45986** [comment]
Alright, @Benefit-Zebra . I've got something. But since all my work is directed towards the manager now, it requires a file from there, snowyPowerup.py.  The two customized files are in this repository, though.

The powerup is called Volcano, and I deviated from what you said, but the concept is still there.  When the powerup is used, the user will die, but the point will create a red "volcano circle" which will "erupt" several normal bombs. Additionally, an impact bomb will fall from the sky onto every other player.  I adjusted it so it'll leave someone with a little health from full, but if they have less than that, they die and the user gets the points.

It offers the same benefits as the curse but is intentionally rare, seeing as it is a quite odd, possibly not very helpful powerup. However, I truly like the idea, and it was fun to make and test.  Keep the good ideas coming...

You're still going to have to describe what you want for that other powerup.

**Comment by Gengar-lab** [comment]
@MattZ45986 I thank you for making my suggested powerup .Please include it in Official Bombsquad mod manager so that others can also use it.

I liked that you improved the idea of powerup.

For other powerup;

I was saying that when we take this powerup , the health of player would increase than normal but only for once until he dies. 

If you think this idea is good then only make this. If you don't liked it say No, It would be Okay for Me.

I would also like to know what you are current working on.

New Powerup:

Name : Destiny bond

Ability : The player (who took the powerup) and then died.This powerup will curse the last player/bot who touched the player.

Special : 1) It will stay activated until player dies.

2) If Player dies (after taking powerup) and no one has touched him , Any one player (Of opposite team , or anyone in FFA) will get curse.

Please reply.

**Comment by Benefit-Zebra** [comment]
@MattZ45986 Waiting for reply.

**Comment by Gengar-lab** [comment]
@MattZ45986

It has been so long time.Waiting for reply.

**Comment by ZKS12** [comment]
Im not able to use flag day plugin
